pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -106,15 +115,25 @@ stdenv.mkDerivation rec {
     nlohmann_json
     nsync
     re2
-    microsoft-gsl
+    (microsoft-gsl.overrideAttrs (oa: {
+      patches = [
+        (fetchurl {
+          url = "https://raw.githubusercontent.com/microsoft/onnxruntime/v${version}/cmake/patches/gsl/1064.patch";
+          sha256 = "sha256-0kv4z68mBuwrVe7UjTBbe0k7c1EAqm80JFaHrrPm3U0=";
+        })
```

Comment:
let's add a code comment explaining why this override and patch are necessary

Reply:
guess https://github.com/microsoft/onnxruntime/pull/17843

Reply:
actually it's this: https://github.com/microsoft/onnxruntime/pull/13573